mbuild/tests/test_xyz.py

```diff
@@ -15,17 +16,24 @@ def test_load_no_top(self, ethane):
         assert ethane_in.n_bonds == 0
         assert set([child.name for child in ethane_in.children]) == {'C', 'H'}
 
-    def test_load_with_top(self, ethane):
-        ethane.save(filename='ethane.xyz')
-        ethane.save(filename='ethane.mol2')
-        ethane_in = mb.load('ethane.xyz', top='ethane.mol2')
-        assert len(ethane_in.children[0].children) == 8
-        assert ethane_in.n_bonds == 7
-        assert (set([child.name for child in ethane_in.children[0].children]) 
-                == {'C', 'H'})
-
     def test_wrong_n_atoms(self):
         with pytest.raises(MBuildError):
             mb.load(get_fn('too_few_atoms.xyz'))
         with pytest.raises(MBuildError):
             mb.load(get_fn('too_many_atoms.xyz'))
+
+    def test_bad_input(self, ethane):
+        with pytest.raises(ValueError):
+            assert isinstance(ethane, mb.Compound)
+            write_xyz(ethane, 'compound.xyz')
+
+    def test_save(self, ethane):
+        ethane.save(filename='ethane.xyz')
+        ethane_in = mb.load('ethane.xyz')
+        assert len(ethane_in.children) == 8
```

Comment:
maybe we also want to make sure that bonding info isnt being added here?

It should be fine, but just a way to make sure nothing in this workflow actually infers bonds